packages/Sortable/src/Sortable.js

```diff
@@ -66,7 +66,9 @@ const Sortable = ({ children, onChange, hasNumbers, onRemove }) => {
         {(provided, snapshot) => (
           <ul
             {...provided.droppableProps}
+            {...moreProps}
             css={sortableStyles}
+            data-display-name="sortable"
```

Comment:
In addition to https://github.com/acl-services/paprika/issues/101#issuecomment-508814582, this is even more confusing to me because it's on a DOM component. To an outside observer, this looks like some magical way of setting `displayName` on this `<ul>`, which it isn't.